internal/apps/unit-of-work/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM golang:latest
```

Comment:
@ahmed-mez Sorry for the late ping on this, but what's the security benefit of this vs specifying a known version (without sha1 to verify)? Arguably both could be problematic if somebody gains write access to the image repository?

Reply:
It's likely for stability, the `latest` tag can include breaking changes.
I'm pretty sure @ajgajg1134 was following an internal guide to make these changes, Andrew feel free to chime in.

Reply:
Whups, I meant to ping @ajgajg1134 🙈